cdv/cmds/chia_inspect.py

```diff
@@ -680,8 +676,8 @@ def one_or_zero(value):
 
             if kwargs["synthetic"]:
                 if sk:
-                    sk = calculate_synthetic_secret_key(sk, hexstr_to_bytes(kwargs["hidden_puzhash"]))
-                pk = calculate_synthetic_public_key(pk, hexstr_to_bytes(kwargs["hidden_puzhash"]))
+                    sk = calculate_synthetic_secret_key(sk, bytes32(hexstr_to_bytes(kwargs["hidden_puzhash"])))
```

Comment:
`bytes32.from_hexstr()`

Reply:
that was the linter, but ill fix it while im here :)

Reply:
oh wait im blind. thanks!